contributing/release-schedule.md

```diff
@@ -1,84 +1,154 @@
+
+## Release Cadence
+
+We are intentionally cautious with our release cadence.
 
-## Determining if your change is in production
+In determining how often we should push new versions of AMP to everyone, we have to weigh many factors including:
+
+- stability for the millions of sites/billions of pages built using AMP
+- cache busting that might happen when we push a new version
+- the desire to get new features out quickly
+
+After considering all of these factors we have arrived at the 1-2 week push cycle. Thus far we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.
+
+
+### Detailed schedule
 
-You can determine what is in a given build using one of the following:
+We try to stick to this schedule as closely as possible, though complications may cause delays. You can track the latest status about any release in the [_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) and the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
 
-- The ["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) which lists the PRs in that release.
-  - The most recent "Type: Release" issue with `(Production)` in the title is the build currently in production.
-  - The "Type: Release" issue with `(Canary)` in the title tracks the current canary.
-- The [PR Use: In Canary](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22) and [PR Use: In Production](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22) labels are added to PRs when they've made it into a canary/production build. There may be a delay between when the build is created and when it goes live.
+- Tuesday @ [11am Pacific](https://www.google.com/search?q=11am+pacific+in+current+time+zone): a new canary release build is created from the [latest master build that passes all of our tests](https://travis-ci.org/ampproject/amphtml/branches) and is pushed to users of AMP who opted into the [AMP Dev Channel](#amp-dev-channel)
+- Wednesday: we check bug reports for _Dev Channel_ users and if everything looks fine, we push the canary to 1% of AMP pages
+- Thursday-Monday: we continue to monitor error rates and bug reports for _Dev Channel_ users and the 1% of pages with the **experimental**/**beta** builds
+- Tuesday (about a week after the **beta** release build was cut): the **beta** build is fully promoted to **stable** (i.e. all AMP pages will now use this build)
+TODO: Flow chart/diagrams and explanations
 
-## Cherry picks
+## Cherry-picks
```

Comment:
I feel like this section should be in a separate doc

Reply:
Moved into `contributing-code.md`